### ISSUE TITLE
有新的开发文档吗？

### ISSUE THREAD

**Issue by windpoiuytr** [a-schild/jave2#254]
有新的开发文档吗？还是说旧的开发文档就可以？

**Comment by windpoiuytr** [comment]
Is there any new development documentation?

**Comment by a-schild** [comment]
No, but I would gladly accept an enhanced/updated documentation

**Comment by a-schild** [comment]
Closing this as part of a tidy up before 4.0.0.

This is a usage question rather than a defect, and it has been sitting unanswered long enough that the answer would likely be out of date anyway. The bundled ffmpeg has moved from 4.4 to 9.0.1 on every platform since it was asked.

Better places for this now:

- the wiki, https://github.com/a-schild/jave2/wiki
- the examples, https://github.com/a-schild/jave2/blob/master/Examples.md

**If you still need an answer, please reopen** or open a new issue with what you are trying to do and what you tried. Questions are welcome, this is only about not leaving old ones open indefinitely.
